lib/types.d.ts

```diff
@@ -0,0 +1,13 @@
+import * as eslint from 'eslint';
+import * as estree from 'estree';
```

Comment:
please enable synthetic imports and esmoduleinterop, TS's module system is broken otherwise - `import * as ` is incorrect here.

Reply:
I had enabled esmoduleinterop. Does the import statements need to be changed? I looked into the `@types/eslint/index.d.ts` file, and it has name exports.

Reply:
`import eslint from 'eslint'` is the proper form; `import * as` should almost never be used.

Reply:
Changed to default import.

Reply:
```suggestion
import estree from 'estree';
```